src/platform/common/platform/fileSystem.ts

```diff
@@ -16,15 +16,21 @@ const ENCODING = 'utf8';
 @injectable()
 export class FileSystem implements IFileSystem {
     protected vscfs: vscode.FileSystem;
+    public rootDirectory: string;
+
     constructor() {
         this.vscfs = vscode.workspace.fs;
+        this.rootDirectory = vscode.Uri.file('/').path;
+    }
+
+    public normalize(path: string): vscode.Uri {
```

Comment:
Since it's wrapping `vscode.Uri.file`, I would suggest we call it `file`. `Normalize` is a bit ambigious here.